src/scenario.py

```diff
@@ -537,34 +537,36 @@ def run(self, strategy_name, options):
                     # write row to file
                     timeseries_file.write('\n' + ','.join(map(lambda x: str(x), row)))
 
+        # calculate results
+
+        sum_cs = []
+        xlabels = []
+
+        for r in results:
+            xlabels.append(r['current_time'])
+            cur_cs = []
+            for cs_id in sorted(self.constants.charging_stations):
+                cur_cs.append(r['commands'].get(cs_id, 0.0))
+            sum_cs.append(cur_cs)
+
+        # untangle external loads (with feed-in)
+        loads = {}
+        for i, step in enumerate(extLoads):
+            for k, v in step.items():
+                if k not in loads:
+                    # new key, not present before
+                    loads[k] = [0] * i
+                loads[k].append(v)
+            for k in loads.keys():
+                if k not in step:
+                    # old key not in current step
+                    loads[k].append(0)
+
```

Comment:
Why is this outside of the visual check? It does not seem to be used if you're not planning to plot. In the test_scenario file, I found only max_total_load to be used.